src/garage/envs/multi_task_env.py

```diff
@@ -0,0 +1,139 @@
+"""
+A wrapper env that handles multiple tasks from different envs.
+
+Useful while training multi-task reinforcement learning algorithms.
+"""
+
+import akro
+import numpy as np
+
+from garage.envs import GarageEnv
+
+
+class MultiTaskEnv(GarageEnv):
+    """
+    A wrapper class to handle multi-task environments.
+
+    Args:
+        task_env_cls_dict (dict(str, garage.envs.GarageEnv)):
```

Comment:
This design is hard to read and shouldn't be necessary with pickling. I think we should just pass a list of objects implementing `gym.Env`.

@zhanpenghe  @krzentner WDYT?